Gemfile.lock

```diff
@@ -115,11 +111,13 @@ GEM
       unf (>= 0.0.5, < 1.0.0)
     dor-rights-auth (1.3.0)
       nokogiri
-    dor-services (5.32.1)
-      active-fedora (>= 7.0, < 9.a)
+    dor-services (6.1.11)
```

Comment:
I wonder what is blocking this from 6.8.0?

Reply:
Ah, I bet it's the pin of dor-services-client

Reply:
i am chasing my own tail here - it looks like we currently need `assembly-utils` but the latest version v1.5 doesn't pin `dor-services` to earlier than 6

trying to update assembly-utils to the latest version here, but failing...